.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "features/cucumber-tck"]
 	path = features/cucumber-tck
-	url = git://github.com/cucumber/cucumber-tck.git
+	url = https://github.com/konserw/cucumber-tck.git 
```

Comment:
This is to upgrade to the latest Cucumber/RSpec/Aruba, right? I'll create a branch in `cucumber-tck` for that so that we can reference an official branch. I also think this deserves its own PR.